frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/ScopeList/ScopeList.tsx

```diff
@@ -23,3 +39,80 @@ export const ScopeList = (): ReactElement => {
     </StudioAlert>
   );
 };
+
+type ScopeListContentProps = {
+  allScopes: MaskinportenScope[];
```

Comment:
I suggest renaming this variable to maskinportenScopes, as allScopes represents a combination of scopes from Maskinporten and our database. 😊 

```suggestion
  maskinportenScopes: MaskinportenScope[];
```